Automattic/woocommerce-payments
# feat: UPE preview onboarding second step

Helps with https://github.com/Automattic/woocommerce-payments/issues/2170

#### Changes proposed in this Pull Request










I decided to split a bigger PR. Other steps of the onboarding flow will be done separately.
This just makes some UI changes on the second onboarding step.

![2021-07-16 13 46 07](https://user-images.githubusercontent.com/273592/125994808-6c8a6800-c123-49ca-84f7-6cabc250ded4.gif)



#### Testing instructions





- Ensure you have `_wcpay_feature_account_overview_task_list` with `'1'` in your `wp_options`
- Ensure "Enable UPE settings changes" is enabled
- Ensure "Enable UPE checkout" is disabled
- Ensure you have "Enable Customer multi-currency" enabled
- Go to http://localhost:8082/wp-admin/admin.php?page=wc-admin&task=woocommerce-payments--additional-payment-methods
- Click "Enable"
- Second step is added and allows to select the payment methods
  - By requirements, all the payment methods are enabled by default
  - A notice is displayed if the store's currency is not EUR and an EUR payment method is selected

If the "multi currency" flag is disabled, you'll get a notice.
But we'll launch at the same time, so no biggie.
![Screen Shot 2021-07-16 at 1 42 01 PM](https://user-images.githubusercontent.com/273592/125994362-517c4716-c483-4236-82e1-b756cab7d7a3.png)

-------------------

Tested in mobile, but looks like 💩 on small screens, just like the "Add payment methods" modal. Separate ticket for that: https://github.com/Automattic/woocommerce-payments/issues/2466
